### ISSUE TITLE
PHPUnit Adapter / Accessibility: low contrast on light terminal

### ISSUE THREAD

**Issue by alexmartinfr** [nunomaduro/collision#123]
**Problem:**
When using a terminal with light color preset*, some contrast issues appears.

- The test suite run summary is using bright ANSI colors, which are hard / impossible to read on a white background:

<img width="384" alt="Capture d’écran 2020-06-07 à 14 42 41" src="https://user-images.githubusercontent.com/19224681/83972282-8d4c9080-a8df-11ea-82d4-054efe4f9e4c.png">

**Suggested solution:**
Here is what it could look like with normal yellow & green ANSI colors, and black for the _Tests_ & _Time_ labels:

<img width="363" alt="Capture d’écran 2020-06-07 à 16 24 29" src="https://user-images.githubusercontent.com/19224681/83972285-8f165400-a8df-11ea-9bd0-f92347f8e0ab.png">

---
**Problem:**
- The PASS label colors aren't contrasted enough on a terminal with light background. It makes them hard to read (WARN label's colors are almost OK according to the WCAG test, but still look wrong):

![CollisionPestActual](https://user-images.githubusercontent.com/19224681/83975416-2ab1bf80-a8f4-11ea-9537-95515e69ffb2.png)

The Jest framework use the terminal's background color as foreground color for its text label, which looks better at first sight. This actually increase contrast with a black background, however it lowers it with a white background, especially for the WARN label which becomes even harder to read. Here is what it would look like for Collision with PHPUnit / Pest:

![CollisionPestJest-like](https://user-images.githubusercontent.com/19224681/83975417-2d141980-a8f4-11ea-8467-736ced109499.png)

**Suggested solution:**
A more interesting all-around solution would be to use bright ANSI colors as label background everywhere. This is not the solution providing the better contrast in every situation, but it has the advantage of displaying a more unified contrast overall and improving both WARN & PASS :

![CollisionPestHarmonized](https://user-images.githubusercontent.com/19224681/83975418-2dacb000-a8f4-11ea-80f8-76ff0be492a1.png)

Only drawback is that it lowers FAIL contrast, but it still hover around the WCAG 3 target for big text. (Plus, the actual shade of red will depend on the terminal theme).

---

_* These examples use iTerm 2 with the default Light background color preset, and [Snazzy](https://github.com/sindresorhus/iterm2-snazzy) as the dark color preset_.
